chroma_core/services/job_scheduler/job_scheduler.py

```diff
@@ -1741,3 +1742,31 @@ def update_lnet_configuration(self, lnet_configuration_list):
     @property
     def CommandPlan(self):
         return CommandPlan(self._lock_cache, self._job_collection)
+
+    def configure_stratagem(self, stratagem_data):
+        with self._lock:
+            with transaction.atomic():
+                StratagemConfiguration.objects.all().delete()
```

Comment:
Why delete everything here?

Reply:
My thought was that anytime we receive a request to configure stratagem we would delete the old configuration and then define the new configuration. I can try to create a new entry where the id is 1 and see if it overwrites the existing values in the db. If that works I should be able to remove the delete.

Reply:
You can just update the existing object. It's equivalent to an `UPDATE` in SQL.

Reply:
Updated to the following:
```
configuration_data = {
                    'state': "unconfigured",
                    'id': 1,
                    'interval': stratagem_data.get('interval'),
                    'report_duration': stratagem_data.get('report_duration'),
                    'report_duration_active': stratagem_data.get('report_duration_active'),
                    'purge_duration': stratagem_data.get('purge_duration'),
                    'purge_duration_active': stratagem_data.get('purge_duration_active')
                }

                if StratagemConfiguration.objects.exists() == True:
                    stratagem_configuration = StratagemConfiguration.objects.update(**configuration_data)
                else:
                    stratagem_configuration = StratagemConfiguration.objects.create(**configuration_data)
```